Makefile

```diff
@@ -13,7 +13,9 @@ build:
 
 serve:
 	$(MAKE) build
-	docker-compose up -d
+	docker-compose up -d db
+	./mysql/bin/wait_for_mysql
+	docker-compose up
```

Comment:
Looks like the jenkinsfile is using this command at the moment, and the lack of a `-d` flag is causing it to hang.